lib/install/create_hooks.dart

```diff
@@ -60,10 +60,12 @@ class CreateHooks {
     var commandPath = '';
     await _hooksCommand((hookFile) async {
       var hookTemplate = hookFile.readAsStringSync();
-      var match =
-          RegExp(r'dart\s(\S+)\s\$hookName').allMatches(hookTemplate).first;
-      commandPath = match.group(1);
-      return false;
+      var match = RegExp(r'dart\s(\S+)\s\$hookName').firstMatch(hookTemplate);
+      if (match is RegExpMatch) {
+        commandPath = match.group(1)!;
+        return false;
+      }
+      return true;
```

Comment:
Changed this to use `firstMatch()` instead of `allMatches()` which is the same and will return null if it didn't match any. The old way `allMatches().first` will throw a `StateError` if there is no match.